app/views/shared/_form_fields.html.erb

```diff
@@ -1,12 +1,11 @@
-<div class="form-group">
-  <%= f.label :title %>
+<%= render layout: "shared/form_group", locals: { f: f, field: :title } do %>
   <%= f.text_field :title, class: 'form-control' %>
-</div>
-<div class="form-group">
-  <%= f.label :summary %>
+<% end %>
+
+<%= render layout: "shared/form_group", locals: { f: f, field: :summary } do %>
   <%= f.text_area :summary, class: 'form-control short-textarea' %>
-</div>
-<div class="form-group">
-  <%= f.label :body %>
+<% end %>
+
+<%= render layout: "shared/form_group", locals: { f: f, field: :body } do %>
```

Comment:
`render layout:` felt like the cleanest way of passing a yield-able block to a template. Unless that can be done in a cleaner way with partials?